rcl/test/rcl/test_init.cpp

```diff
@@ -98,9 +98,9 @@ struct FakeTestArgv
   FakeTestArgv(const FakeTestArgv &) = delete;
 };
 
-/* Tests the rcl_init(), rcl_ok(), and rcl_shutdown() functions.
+/* Tests the rcl_init() and rcl_shutdown() functions.
  */
-TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_init_and_ok_and_shutdown) {
+TEST_F(CLASSNAME(TestRCLFixture, RMW_IMPLEMENTATION), test_rcl_init_and_shutdown) {
```

Comment:
This is also testing `rcl_context_is_valid` which replaced `rcl_ok`, though I don't have a better name for the test case.

Reply:
Yeah, I'm not sure about the name either. While this test happens to be checking `rcl_context_is_valid`, IMO, any unit tests specifically targeting that function should go in `test_context.c` anyways, so I'll leave this as-is.